.codespellrc

```diff
@@ -0,0 +1,8 @@
+[codespell]
+check-filenames =
+check-hidden =
+count =
+# Disable warnings about binary files
+quiet-level = 2
+skip = */.git,*/common_tools/cloc
+ignore-words-list = thur,inout,te,nd,lod,aci,nin,nnumber,wile,reall,somwhere
```

Comment:
I think I had changed them in a previous commit and then decided that this correction wasn't necessarily a spelling error since maybe `somwhere.com` is as valid as `somewhere.com` as an email address/domain name.  Similar to not changing `bogous` to `bogus`  -- maybe was intentionally not a valid word...

Reply:
It may be a valid domain name, but I think the intention was clearly `somewhere.com`. I think we should fix it. @bartlettroscoe WDYT?

Reply:
Where is somwhere flagged?  I don't think that is valid name.

Reply:
This is fixed in commit [`d898491` (#573)](https://github.com/TriBITSPub/TriBITS/pull/573/commits/d898491b733902e90f261d7dddf1adbc848637da).